src/gamma.jl

```diff
@@ -100,13 +100,13 @@ function cotderiv_q(m::Int)
     q₋ = cotderiv_q(m-1)
     d = length(q₋) - 1 # degree of q₋
     if isodd(m-1)
-        q = Array{Float64}(length(q₋))
+        q = Vector{Float64}(uninitialized, length(q₋))
```

Comment:
needs higher min compat - is this in a release yet?

Reply:
yeah